src/HDInsight/HDInsight/ManagementCommands/GrantAzureHDInsightRdpServicesAccessCommand.cs

```diff
@@ -16,11 +16,13 @@
 using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
 using Microsoft.Azure.Management.HDInsight.Models;
 using Microsoft.WindowsAzure.Commands.Common;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 using System;
 using System.Management.Automation;
 
 namespace Microsoft.Azure.Commands.HDInsight
 {
+    [CmdletDeprecation()]
```

Comment:
Please provide with an instruction for the customers who are using the cmdlet on how they should do with their jobs and scripts.

Reply:
> Please provide with an instruction for the customers who are using the cmdlet on how they should do with their jobs and scripts.

Hi @msJinLei , I think that we don't have to do this. There is no replacement of this cmdlet.  This cmdlet is related with specific os type cluster which has been deprecated. Customers can't use this cmdlet any more. We have removed this cmdlet in previous PR #9941 .

Reply:
> > Please provide with an instruction for the customers who are using the cmdlet on how they should do with their jobs and scripts.
> 
> Hi @msJinLei , I think that we don't have to do this. There is no replacement of this cmdlet. This cmdlet is related with specific os type cluster which has been deprecated. Customers can't use this cmdlet any more. We have removed this cmdlet in previous PR #9941 .

when customer use this cmdlet they will get a message: "The cmdlet is being deprecated. There will be no replacement for it." This is actually a good instruction.